docs/content/users/quickstart.md

```diff
@@ -149,6 +149,40 @@ While the generic `php` project type is [ready to go](./project.md) with any CMS
         ddev launch
         ```
 
+=== "ExpressionEngine"
+
+    ## ExpressionEngine
+
+    === "ExpressionEngine Zipfile Download"
+
+        Download the ExpressionEngine code from [ExpressionEngine.com](https://expressionengine.com/), then:
+
+        ```bash
+        mkdir my-ee && cd my-ee
+        unzip /path/to/ee-zipfile.zip
+        ddev config # Accept the defaults
+        ddev start
+        ddev launch /admin.php
+        ```
+
+        In the `admin.php` setup, use "db" for the "DB Server Address", "DB Name", "DB Username", and "DB Password"
```

Comment:
Continue with the “installation wizard” language from above for context, use backticks to signify a literal string and (hopefully) avoid any confusion with quotes, and italicize UI strings for consistency with treatment elsewhere in the docs:

```suggestion
        When the installation wizard prompts for database settings, enter `db` for the _DB Server Address_, _DB Name_, _DB Username_, and _DB Password_.
```